### DISCUSSION TITLE
Welcome to mcuboot Discussions!

### DISCUSSION THREAD

**Discussion by utzig** [mcu-tools/mcuboot#919] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nvlsianpu** [comment]
Hi.

**Comment by d3zd3z** [comment]
I wonder if this makes a good place to discuss RFC-type things.
